### ISSUE TITLE
`transform` and `embed` costs are 1, and should be 10M and 1M respectively

### ISSUE THREAD

**Issue by montanalow** [postgresml/postgresml#755]
The planner is not using these functions appropriately.

**Comment by cybro-rob** [comment]
I'm a first timer, can I contribute?

**Comment by Tostino** [comment]
Just wanted to show where this causes issues:
Good (runs for just the expected rows):
```
SELECT pgml.transform(
    'summarization',
    inputs => ARRAY[cleaned_transcript]
) AS result
FROM (SELECT left(cleaned_transcript, 5000) cleaned_transcript FROM youtube_transcripts WHERE cleaned_transcript IS NOT NULL ORDER BY length(cleaned_transcript) LIMIT 10) x;
```
Bad (runs function for all rows prior to applying the LIMIT):
```
SELECT pgml.transform(
    'summarization',
    inputs => ARRAY[left(cleaned_transcript, 5000)]
) AS result
FROM youtube_transcripts WHERE cleaned_transcript IS NOT NULL ORDER BY length(cleaned_transcript) LIMIT 10;
```

**Comment by montanalow** [comment]
That'd be great @cybro-rob, I think we just need a sql migration file for the next version to fix this.

**Comment by montanalow** [comment]
Fixed in 2.7.0
